server/store/storemodels/storemodels.go

```diff
@@ -9,6 +9,11 @@ const (
 	PreferenceNamePlatform         = "platform"
 	PreferenceValuePlatformMM      = "mattermost"
 	PreferenceValuePlatformMSTeams = "msteams"
+
+	// Notification preferences
+	PreferenceNameNotification     = "notifications"
+	PreferenceValueNotificationOn  = "1"
```

Comment:
nit: why `1` and `0` instead of `on` and `off`?

Reply:
I swear I saw some preference using `1`/`0`  but can't find them anymore haha, will switch to `on`/`off` which is what I had initially.